save_parameter.php

```diff
@@ -1,7 +1,7 @@
 <?php
 
 $base_dir = dirname( __FILE__ ) . '/../../../';
-require( $base_dir . 'wp-load.php' );
+require( $base_dir . 'wp/wp-load.php' );
```

Comment:
This would only work with subdirectory installs. Guess a more generic solution should be implemented as the current solution doesn't take subdirectory installs into account.